cloudnative-pg.yaml

```diff
@@ -31,15 +20,15 @@ pipeline:
   - uses: go/build
     with:
       modroot: ./cmd/manager
-      output: manager_${{vars.mangled-arch}}
+      output: manager_${{build.goarch}}
```

Comment:
@imjasonh But this won't made the replacement from 'aarch64' to 'arm64', afaik.

Reply:
I believe it will -- we can see it's named correctly in the package:

```
$ make package/cloudnative-pg
$ tar -tvf packages/aarch64/cloudnative-pg-1.23.2-r6.apk
...
-rwxr-xr-x  0 jason  dialout 58386910 Jul  5 10:44 bin/manager_arm64
lrwxr-xr-x  0 jason  dialout        0 Jul  5 10:44 manager -> /bin/manager_arm64
```